sfge/src/main/java/com/salesforce/graph/vertex/FieldWithModifierVertex.java

```diff
@@ -33,4 +33,8 @@ public boolean isPrivate() {
     public boolean isStatic() {
         return getModifierNode().isStatic();
     }
+
+    public boolean isAbstract() {
```

Comment:
Would this be more fitting in MethodVertex? `abstract` keyword wouldn't be used with field definitions.